Binary files orig/jpeg/src/.Makevars-ls.win.swp and patched/jpeg/src/.Makevars-ls.win.swp differ
diff -Nru orig/jpeg/src/Makevars-ls.win patched/jpeg/src/Makevars-ls.win
--- orig/jpeg/src/Makevars-ls.win	2022-11-29 10:23:07.000000000 +0100
+++ patched/jpeg/src/Makevars-ls.win	2023-10-10 09:37:52.363223300 +0200
@@ -1,2 +1,7 @@
 ## This Makevars is used on Windows when system jpeg is to be used
-PKG_LIBS=-ljpeg
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_LIBS=-ljpeg
+else
+  PKG_LIBS=$(shell pkg-config --libs libjpeg)
+endif
+
